lists.js

```diff
@@ -398,6 +398,20 @@ List.prototype.becomeLinked = function () {
     }
 };
 
+List.prototype.toJSON = function () {
+    return this.asArray();
```

Comment:
This should be `this.itemsArray();`, I think. So we don't convert linked lists to arrays.